src/Cache/Invalidation.php

```diff
@@ -570,8 +582,7 @@ public function on_transition_post_status_cb( $new_status, $old_status, WP_Post
 			$action_type = 'CREATE';
 		}
 
-		$post_type_object = get_post_type_object( $post->post_type );
-		$type_name        = $post_type_object instanceof \WP_Post_Type ? strtolower( $post_type_object->graphql_single_name ) : $post_type_object;
+		$type_name = strtolower( $post_type_object->graphql_single_name );
```

Comment:
So with changes like this we _really_ need to be careful that we're only executing code when WPGraphQL is active. 

If WPGraphQL Smart Cache were active but WPGraphQL was not active, and the invalidation callbacks were fired, I think this would be some fatals. 

So, we should check to ensure we're only executing code if WPGraphQL is already active. 👀

Reply:
Ok, I confirmed this is the case: https://github.com/wp-graphql/wp-graphql-smart-cache/blob/main/wp-graphql-smart-cache.php#L191-L210